Perl/syntax_test_perl.pl

```diff
@@ -869,7 +904,7 @@ =head1 B<--param>
 # <- meta.string.heredoc.perl text.html.embedded.perl
 #^^^^^^^^^ meta.string.heredoc.perl text.html.embedded.perl
    HTML
-#  ^^^^ meta.string.heredoc.perl text.html.embedded.perl - constant.other.language-name.html.perl
+#  ^^^^ meta.string.heredoc.perl text.html.embedded.perl - constant.other.language-name.perl
```

Comment:
This negation still feels oddly specific to me. Did you mean for `- constant` or `- constant.other` instead?

Reply:
That was too long ago to remember. This line is just a result of search and ~~destroy~~ replace.